core/modules/replica/SemanticMaps.h

```diff
@@ -394,6 +406,137 @@ using WorkerChunkDatabaseMap =
      }
  }
 
+/**
+ * One-directional comparison of dictionaries of: worker-database-chunk
+ *
+ * The method will also report keys which aren't found in second dictionary.
+ *
+ * NOTE: the output dictionary will be modified even if the method
+ *       will not find any differences.
+ *
+ * @param one       - input dictionary to be compared with the second one
+ * @param two       - input dictionary to be compared with the first one
+ * @param inOneOnly - output dictionary with elements of the first map which
```

Comment:
I'd probably call it `inFirstOnly`, `inOneOnly` to me reads as `in any one only`

Reply:
👍